pkg/webhook/webhook.go

```diff
@@ -99,6 +115,37 @@ func (s *Server) handleWebhook() http.Handler {
 
 		if err = s.eventsPubsub.Send(context.Background(), eventBytes); err != nil {
 			logger.Errorw("failed to write messages to event pubsub", "code", http.StatusInternalServerError, "body", errWritingToBackend, "error", err)
+
+			exceeds, bqQueryErr := s.bqClient.
+				FailureEventsExceedsRetryLimit(r.Context(), s.failureEventTableID, deliveryID, s.retryLimit)
```

Comment:
Nit: It would be better to pull `r.Context()` into a `ctx` variable closer to the top.

Reply:
ack